README.md

```diff
@@ -0,0 +1,42 @@
+# Kickstart
+
+Use Kickstart so you don't have to remember the names of your scripts!
+
+# Contents
+
+- [Description](#description)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Description
+
+When you're working on a project with multiple people you might've noticed that everyone is using different script names for the most common tasks. Where one might use `npm start` for starting the development mode of an application, someone else is using that command for starting the production build. Especially when you're working on more than one application you might have a hard time remembering all the right commands.
```

Comment:
Aren't multiple people in the same project forced to use the same script names?

```suggestion
When you're working on more than one application you might have noticed that more often than not different script names are used for the most common tasks. Where one might use `npm start` for starting the development mode of an application, someone else is using that command for starting the production build.
```

Reply:
I'd try to come up with an example where it's most likely to encounter mismatches. So I though of a project you're working on together, then it's very common the standards of each regarding script names differ. Working on different applications that are all your own won't cause that situation 😅

Reply:
Ah! I meant multiple applications with multiple people. Edited the suggestion. I interpret the current version as multiple people working on one application with one set of scripts

Reply:
I've combined the two now, thanks!